bitcoin/src/psbt/serialize.rs

```diff
@@ -206,7 +206,7 @@ impl Serialize for KeySource {
 
         rv.append(&mut self.0.to_byte_array().to_vec());
 
-        for cnum in self.1.into_iter() {
+        for cnum in &self.1 {
```

Comment:
FTR, this looks like semantics change but it's not. `self.1` is a wrapped `Vec` and `self` is behind reference so `.` operator was previously doing autoref.

Reply:
Yeah, autoref and `into_iter` is a great combination for writing obfuscated Rust.